testing/services/pkg/deployment/services.go

```diff
@@ -84,10 +94,18 @@ func (config *TestConfig) StartServiceTest(ctx context.Context, clientset *kuber
 		err = config.DualStackDeployment(ctx, clientset)
 		if err != nil {
 			slog.Error(err)
+			errs = append(errs, err)
 		}
 	}
 
-	slog.Infof("🏆 Testing Complete [%d] passed", config.SuccessCounter)
+	printf := slog.Infof
+	if len(errs) > 0 {
+		printf = slog.Errorf
+	}
+
+	printf("🏆 Testing Complete [%d] passed / [%d] failed", config.SuccessCounter, len(errs))
```

Comment:
use a constant for the string. and call the logger instead of indirect assignment of different log functions to `printf`